app/assets/javascripts/covid19/src/webapp/components/structures-table/Columns.tsx

```diff
@@ -80,7 +81,16 @@ export const columnsBase: Columns = [
         width: 180,
         sortable: false,
         renderCell: OrganismCell,
-        renderString: row => row.organisms.map(organism => organism.name).join(", "),
+        renderString: row =>
+            row.organisms
+                .map(
+                    organism =>
+                        organism.name +
+                        (organism.commonName || organism.commonName === "?"
```

Comment:
Not sure about the logic. `value || value === something` -> `value === something` ? or what was the idea?

Reply:
It's because there are some organisms with common name "?". So if the common name is undefined or equals '?', then it will return an empty string.

Reply:
So you mean `!organism.commonName` ?

Reply:
Not exaclty. I mean, yes; but also if doesn't equal '?'.

Reply:
Or should it export the commonName even if it's '?'...?

Reply:
I was only referring to the first check, `!organism.commonName || organism.commonName === "?"` looks good

Reply:
Oh sorry, that's true. I didn't notice it. I apologize.